scss/variables/typography.js

```diff
@@ -10,7 +10,8 @@
 
 var color = "#222",
 	fontSize = "20px",
-	lineHeight = 1.5;
+	lineHeight = 1.5,
+	linkColor = "#4078c0";
 
```

Comment:
Ideally these colors should come from a palette defined somewhere? We could have say shades of Blue as the Blue palette and use different shades for different states of a link.